kolibri/utils/sanity_checks.py

```diff
@@ -60,3 +69,28 @@ def check_content_directory_exists_and_writable():
             "The content directory {} is not writable.".format(content_directory)
         )
         sys.exit(1)
+
+
+def check_log_file_location():
```

Comment:
This is good - can you add a follow up issue to migrate this to a version specific upgrade once my PR (#5494) to allow that is merged (for 0.13.0)?

Reply:
yes created it here: https://github.com/learningequality/kolibri/issues/5532